common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -564,7 +564,7 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
     REPL_ADD_RAW_RESERVED_NAMESPACE("hive.repl.add.raw.reserved.namespace", false,
         "For TDE with same encryption keys on source and target, allow Distcp super user to access \n"
             + "the raw bytes from filesystem without decrypting on source and then encrypting on target."),
-    REPL_INCLUDE_EXTERNAL_TABLES("hive.repl.include.external.tables", false,
+    REPL_INCLUDE_EXTERNAL_TABLES("hive.repl.include.external.tables", true,
```

Comment:
nit: Should we do cleanup and remove the usage of ReplicationTestUtils.includeExternalTableClause() with true value passed?
ReplicationTestUtils.includeExternalTableClause(true);